dashboard/src/shared/Operators.ts

```diff
@@ -95,4 +95,65 @@ export class Operators {
     );
     return data;
   }
+
+  public static async createOperator(
+    namespace: string,
+    name: string,
+    channel: string,
+    installPlanApproval: string,
+    csv: string,
+  ) {
+    // First create the OperatorGroup if needed
+    await this.createOperatorGroupIfNotExists(namespace);
+    // Now create the subscription
+    const { data: result } = await axiosWithAuth.post<IResource>(
+      urls.api.operators.subscription(namespace, name),
```

Comment:
Something to think about longer term, rather than for this PR: is the fact that we're interacting with the k8s API directly from the dashboard for all these calls going to be something which helps or hinders us when we start looking at multi-cluster support? (or neither even). If we are able to run kubeops in the one cluster where Kubeapps is installed only, and have it operate with the users credentials on multiple clusters, then the dashboard would only ever interact with kubeops. If on the other hand, we expect the dashboard to be interacting directly with the k8s API, then some functional awareness of the dashboard to determine which cluster is necessary. It may be easy enough do a mixture of both if needed.

Perhaps interacting directly with the k8s API would make it easier to extract functionality into plugins. When using an authProxy, I've no problem at all security-wise with doing this, but when using tokens stored in the browser client it's a bit of a worry. Thoughts?

Reply:
IMO I would rather not expose the whole Kubernetes API and use kubeops to return only the necessary information, although I know that cause a higher bar to implement new functionality (and it's why I am using it for the Operators stuff). If we have the time to refactor that, it'd be great.

Reply:
I don't think that we'll ever be given the time to do things like this - especially if we keep adding to it. If we're considering it as tech-debt, let's not add it in the first place (and consider updating it with any changes to the operator support work, or consider it part of moving the operator work out of alpha/beta).